internal/sync/configs/olm-config.yaml

```diff
@@ -0,0 +1,28 @@
+---
+- name: olm
+  sync: true
+  repo_link: "https://github.com/operator-framework/api"
+  branch: v0.26.0
+  remote_api_directory: pkg/operators/v1alpha1
+  local_api_directory: schemes/olm/v1alpha1
+  excludes:
+    - "*_test.go"
+
+- name: olm
+  sync: true
+  repo_link: "https://github.com/operator-framework/api"
+  branch: v0.26.0
+  remote_api_directory: pkg/operators/v1
+  local_api_directory: schemes/olm/v1
```

Comment:
I think it could use some replaces for:
```
opregistry "github.com/operator-framework/operator-registry/pkg/registry"

operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
```